jaxrs-api/pom.xml

```diff
@@ -182,6 +182,7 @@
                         </plugin>
                         <plugin>
                             <artifactId>maven-surefire-plugin</artifactId>
+                            <version>2.21.0</version>
```

Comment:
Why not 2.22.2?

Reply:
Because it works just fine with 2.21.0 and haven't tried 2.22.2?

Reply:
I agree it was a really bad Idea to use a pre-release of Surefure, but as 2.22.2 is just a bug fix I assume it will work just as good. I would kindly ask you to actually try it out and use that one instead of needlessly falling back to an older version of that plugin, as it definitively *has* more bugs -- that's why dot-two was released. :-)

Reply:
It's really unnecessary as this version works just fine for us and I don't want to delay this any longer. But I'll do it if this is a blocker for _you_.

Reply:
OK, just for the record it _fails_ for me locally with 2.22.2. Others feel free to try. Marking this conversation as resolved.